<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Timer setting</string>
      </property>
      <widget class="QTimeEdit" name="timerEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>19</hour>
         <minute>10</minute>
         <second>0</second>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
      <widget class="QLineEdit" name="session_count_lineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="study_length_lineEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>50</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="break_length_lineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Start studying at</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>with</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>sessions of</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>(study time - break time, e.g., 50-10)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="start_button">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>START</string>
       </property>
      </widget>
      <widget class="QPushButton" name="default_button">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Todo list</string>
      </property>
      <widget class="QPushButton" name="update_todo_button">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>UPDATE</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="todo_1">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="todo_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>53</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="todo_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="todo_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>120</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="load_todo_txt_button">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Reload txt</string>
       </property>
      </widget>
      <widget class="QPushButton" name="removeButton_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QPushButton" name="removeButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QPushButton" name="removeButton_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QPushButton" name="removeButton_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="todoText_1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="todoText_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="todoText_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="todoText_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="todoText_5">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>150</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="todo_5">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="removeButton_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
